examples/components/tabs/tabs.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import router from 'hoc-little-router'
+import classnames from 'classnames'
+import { Tabs as Component} from '../../../components'
```

Comment:
It should not pass linter :/
And Component is not a good name for examples : KriyaTabs ?